spec/requests/api/v2/notifications_spec.rb

```diff
@@ -153,7 +153,7 @@
         it 'returns a notification group covering all notifications' do
           subject
 
-          notification_ids = user.account.notifications.reload.pluck(:id)
+          notification_ids = user.account.notifications.reload.order(id: :asc).pluck(:id)
```

Comment:
Is the `reload` still necessary? I would imagine (but did not confirm) that with the added `order` call this should always go through to the db.

Reply:
You're right. Updated!